cg/apps/demultiplex/sample_sheet/index.py

```diff
@@ -16,6 +17,7 @@
 INDEX_ONE_PAD_SEQUENCE: str = "AT"
 INDEX_TWO_PAD_SEQUENCE: str = "AC"
 LONG_INDEX_CYCLE_NR: int = 10
+MINIMUM_HAMMING_DISTANCE: int = 3
```

Comment:
Why is this value used? Maybe a comment can be added to clarify it 🙂

Reply:
This was the threshold agreed upon. I am not sure of the specific reasons for this number